### PR TITLE
[test][sagemaker] Include ap-northeast-3 in NO_Px_REGIONS lists

### PR DESCRIPTION
**Pull request by saimidu**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [x] (If applicable) I've documented below the tests I've run on the DLC image
- [x] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [x] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Added `ap-northeast-3` to list of regions with no P2, P3, or P4 instance types

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/huggingface_tensorflow/training/integration/__init__.py [hunk 1]
```diff
@@ -38,10 +38,12 @@
     "cn-northwest-1",
     "eu-south-1",
     "af-south-1",
+    "ap-northeast-3",
 ]
 NO_P3_REGIONS = [
```
Comment (jeet4320) [inline test/sagemaker_tests/huggingface_tensorflow/training/integration/__init__.py:43]: do we need NO_P4_REGIONS for HF in this file?

Comment (saimidu) [reply test/sagemaker_tests/huggingface_tensorflow/training/integration/__init__.py:43]: The NO_P4_REGIONS list is only needed for p4d tests, which are only used for SMDDP/SMDMP tests.

Comment (arjkesh) [reply test/sagemaker_tests/huggingface_tensorflow/training/integration/__init__.py:43]: +1 - could you also explain why we don't need to edit pytorch tests?

Comment (saimidu) [reply test/sagemaker_tests/huggingface_tensorflow/training/integration/__init__.py:43]: The Hugging Face Inference and the Hugging Face PyTorch Training tests already have `ap-northeast-3`. Some of the NO_Px_REGIONS lists seem to already have it, while others don't.

This PR just makes them all consistent.

**Review by arjkesh — APPROVED** [review]

**Pull request merged**